docs/docs/Get-Started/get-started-installation.md

```diff
@@ -6,166 +6,228 @@ slug: /get-started-installation
+```
+(VENV_NAME) PS C:/users/username/langflow-dir>
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `Remove-Item VENV_NAME`.
+  </TabItem>
+  </Tabs>
+  <TabItem value="Linux" label="Linux">
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
+```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
+</details>
 
-</TabItem>
-<TabItem value="pip" label="pip">
 
-```bash
-pip install langflow==1.3.2
-```
+<Tabs groupId="os">
+  <TabItem value="macOS" label="macOS">
 
-</TabItem>
-</Tabs>
+    1. To install Langflow, run the following command.
+    ```bash
+    uv pip install langflow
+    ```
 
-To reinstall Langflow and all of its dependencies, add the `--force-reinstall` flag to the command.
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
+  :::important
+  Windows installations of Langflow OSS require a C++ compiler, such as [Visual Studio](https://visualstudio.microsoft.com/downloads/), that may not be present on your system. If you receive a `C++ Build Tools Required!` error, follow the on-screen prompt to install Microsoft C++ Build Tools, or visit the Visual Studio download link above.
+  :::
 
-```bash
-uv pip install langflow --force-reinstall
-```
+      1. To install Langflow, run the following command.
+      ```bash
+      uv pip install langflow
+      ```
+
+  :::tip
+  If you experience an error from the `webrtcvad` package, run `uv pip install webrtcvad-wheels` in the virtual environment, and then try installing again.
+  :::
+
+
+  </TabItem>
+  <TabItem value="Linux" label="Linux">
+
+      1. To install Langflow, run the following command.
+      ```bash
+      uv pip install langflow
+      ```
 
-</TabItem>
-<TabItem value="pip" label="pip">
+  </TabItem>
+
+</Tabs>
+
+2. After installation, to run Langflow, use the following command:
 
 ```bash
-pip install langflow --force-reinstall
+uv run langflow run
 ```
 
-</TabItem>
-</Tabs>
+3. To confirm that a local Langflow instance starts, go to the default Langflow URL at `http://127.0.0.1:7860`.
 
-### Install optional dependencies for Langflow OSS
+After confirming that Langflow is running, create your first flow with the [Quickstart](/get-started-quickstart).
 
-Langflow OSS provides optional dependency groups that extend its functionality.
+### Manage Langflow OSS versions
```

Comment:
This is a grab-bag of miscellaneous commands. I suggest the following changes:

- Make upgrading its own topic.
- Move the info about installing a specific version to the OSS installation topic. 
- Make reinstalling its own topic and add some context around why/when a reinstallation may be necessary.

Reply:
I will add this section to the upgrade topic task in our project plan.